books/RayTracingTheRestOfYourLife.html

```diff
@@ -2109,10 +2097,31 @@
 
         return emitted
             + srec.attenuation * rec.mat_ptr->scattering_pdf(r, rec, scattered)
-                               * ray_color(scattered, background, world, lights, depth-1)
-                               / pdf_val;
+                               * ray_color(scattered, background, world, lights, depth-1) / pdf_val;
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     }
+
+    ...
+
+    int main() {
+        ...
+        // World
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
+        auto lights = make_shared<hittable_list>();
+        lights->add(make_shared<xz_rect>(213, 343, 227, 332, 554, shared_ptr<material>()));
+        lights->add(make_shared<sphere>(point3(190, 90, 190), 90, shared_ptr<material>()));
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
+
+        auto world = cornell_box();
+
+        color background(0,0,0);
+
+        for (int j ...) {
+            for (int i ...) {
```

Comment:
Not sure if i like this
`for (int j ...)`

would prefer a full write out:
`    for (int j = image_height-1; j >= 0; --j) {`

Reply:
Done.